packages/launchpad/src/signer.ts

```diff
@@ -10,7 +9,17 @@ export interface AccountData {
   readonly pubkey: Uint8Array;
 }
 
-export interface SignResponse {
+export interface AminoSignJob {
+  readonly mode: SignMode.LegacyAminoJson;
+  readonly signDoc: StdSignDoc;
+}
+
+export interface DirectSignJob {
+  readonly mode: SignMode.Direct;
+  readonly signBytes: Uint8Array;
```

Comment:
Why is this encoded bytes? There is no higher level objects we would encode? Or we assume the caller knows how to proto serialize?

We don't require the caller to json/amino serialize the StdSignDoc in AminoSignJob, so I wonder why we have to here. If this is a current technical limitation, that is fine, just add a `FIXME` to this type. If this is a conscious decision as to why this is the proper way to do it, can you please document why?